### DISCUSSION TITLE
how to manually build after 9.14.2

### DISCUSSION THREAD

**Discussion by imiskolee** [directus/directus#14767] [Archive]
### Describe the Bug

can not manually local build

### To Reproduce

1. npm install

### Errors Shown

npm WARN While resolving: directus-monorepo@9.14.2
npm WARN Found: jest@27.5.1
npm WARN node_modules/jest
npm WARN   dev jest@"28.1.2" from the root project
npm WARN   2 more (ts-jest, directus)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer jest@"^27.0.0" from ts-jest@27.1.3
npm WARN node_modules/ts-jest
npm WARN   dev ts-jest@"28.0.5" from the root project
npm WARN   1 more (directus)
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*

npm ERR! A complete log of this run can be found in:

### What version of Directus are you using?

8.14.2

### What version of Node.js are you using?

v16.15.1

### What database are you using?

no

### What browser are you using?

no

### How are you deploying Directus?

locally

**Comment by adanielyan** [comment]
Having the same issue. ```yarn``` worked fine.

**Comment by azrikahar** [comment]
The repo was moved to use [pnpm](https://pnpm.io/) since #14350 in `9.14.2`. 

Do refer to the updated **Running Locally** docs for more info: https://docs.directus.io/contributing/running-locally/

You will need to use `pnpm install` to install the dependencies (which base on the `pnpm-lock.yaml` file in the root directory of this repo), and run `pnpm -r build` to recursively run the build command on all the packages in the workspace.
